pkg/util/bpfutil/bpfexamples.go

```diff
@@ -0,0 +1,1593 @@
+// +build linux
+
+/*
+Copyright © 2019 LoT Authors
```

Comment:
We need to add this as well, 
```
Copyright 2018 Netflix, Inc.
```
https://github.com/iovisor/bpftrace/blob/0c08b1d38a5bc5f9cfae7ff016d3361e364ddf6f/tools/loads.bt#L14-L15

Reply:
@bhavin192 The thing is, all of these examples are Copyright under various different authors. Therefore mentioning just one kind of doesn't make sense.

Reply:
Then have to mention all of them with their respective code snippet